handleData.js

```diff
@@ -9,101 +9,108 @@ function processCharacterData(dataSetSize) {
 }
 
 // Begin Message Definition
-var Message = function(content) {
-    this.timeStamp = new Date();
-    this.content = content;
-}
+class Message {
+    constructor(content) {
+        this.timeStamp = new Date();
+        this.content = content;
+    }
 
-Message.prototype.getFormattedTimestamp = function() {
-    return  "Message sent on: " + 
-            this.timeStamp.getDate() + "/" +
-            (this.timeStamp.getMonth() + 1) + "/" +
-            this.timeStamp.getFullYear() + " @ " +
-            this.timeStamp.getHours() + ":" +
-            this.timeStamp.getMinutes() + ":" +
-            this.timeStamp.getSeconds();
+    getFormattedTimestamp() {
+        return  "Message sent on: " + 
+                this.timeStamp.getDate() + "/" +
+                (this.timeStamp.getMonth() + 1) + "/" +
+                this.timeStamp.getFullYear() + " @ " +
+                this.timeStamp.getHours() + ":" +
+                this.timeStamp.getMinutes() + ":" +
+                this.timeStamp.getSeconds();
+    }
 }
 
 // Begin MessageThread Definition
```

Comment:
Yeah I think most of those comments are, at this point. 
Since they are nicely bundled together in "classes"